language/tools/vm-genesis/src/lib.rs

```diff
@@ -750,6 +766,22 @@ fn verify_genesis_write_set(
     );
 }
 
+/// Generate a dummy genesis `ChangeSet` that allows open publishing. For testing only.
+pub fn generate_dummy_genesis_change_set() -> ChangeSet {
```

Comment:
Would `testing` instead of `dummy` in this name be a better name for it since it's only meant to be used in testing?